docs/en/mobile-instance-registration.rst

```diff
@@ -0,0 +1,245 @@
+.. _mobile-instance-app-initialization-and-registration.rst:
+
+Mobile Application Instance Initialization and Registration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since the registration flows are analogous for a Wallet or Relying Party Instance, these components are henceforth called Mobile Application Instance. Similarly, the Application Provider fulfills the roles of the Wallet Provider and Relying Party backend depending on the registration process.   
```

Comment:
```suggestion
Since the registration flows are analogous for a Wallet or Relying Party Instance, these components are henceforth called Mobile Application Instances. Similarly, the Application Provider fulfills the roles of the Wallet Provider and Relying Party backend depending on the registration process.   
```